vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -467,6 +473,16 @@
 my $StartTime = time;
 my $q         = "not_run";
 
+# Default output file names
+my $packing_file = "$benchmark_name" . ".net";
+my $placement_file = "$benchmark_name" . ".place";
+my $routing_file = "$benchmark_name" . ".route";
+my $setup_timing_report = "report_timing.setup.rpt";
+my $hold_timing_report = "report_timing.hold.rpt";
+my $unconstrained_setup_timing_report = "report_unconstrained_timing.setup.rpt";
+my $unconstrained_hold_timing_report = "report_unconstrained_timing.hold.rpt";
```

Comment:
Seems like these are only used in cmp_full_vs_incr_STA(), we should probably define them within that function scope to avoid polluting the file-scope namespace (which has too many variables already).